content/en/logs/guide/collect-google-cloud-logs-with-push.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Collect Google Cloud Logs with a Push Subscription
+kind: documentation
+further_reading:
+- link: "/logs/explorer/"
+  tag: "Documentation"
+  text: "Learn how to explore your logs"
+- link: "/logs/explorer/#visualize"
+  tag: "Documentation"
+  text: "Perform Log Analytics"
+- link: "/logs/log_configuration/processors"
+  tag: "Documentation"
+  text: "Learn how to process your logs"
+---
+
+<div class="alert alert-danger">
+Datadog recommends forwarding your Google Cloud logs with a <strong>Pull</strong> subscription to leverage compression and batching of events. See <a href="https://docs.datadoghq.com/integrations/google_cloud_platform/#log-collection" target="_blank">Log collection</a> on the Google Cloud integration page for instructions.
+</div>
```

Comment:
Mention Dataflow here.

Reply:
If the customer has VPC setup in their Google Cloud environment, the push subscription cannot access an endpoint outside the VPC subscription. We should mention this both here and the google_cloud_platform page.